app.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express")
+const cors = require("cors")
+// const bodyParser = require("body-parser")
+const port = 3001;
+ 
+const app = express();
+app.use(cors());
+
+
+let aniArray = ["ant", "lion", "tiger", "bear"]
+
+const isAnimalMiddleware = (req, res, next) =>{
+    let input = req.params.species
+    
+  if(aniArray.includes(input)){
+      next();
+    } else {
+        res.json("this animal is not included")
```

Comment:
try sending a response message similar to when you do find the animal you're looking for.